libs/table/cells/LinkCell.tsx

```diff
@@ -14,7 +14,7 @@ export const linkCell =
   ({ value }: Cell<string>) => {
     return (
       <Link
-        className="flex h-full w-full items-center text-sans-semi-md text-default hover:underline"
+        className="flex h-full w-full items-center underline text-sans-semi-md text-secondary hover:text-default"
```

Comment:
Is `hover:bg-raise` to distracting here?

Reply:
I was kinda thinking the same

Reply:
Happy to remove

Reply:
"bg-hover" is too strong, and my concern was that "bg-raise" was inconsistent.

My rationale for more actions and the link cell having a different treatment is partly the distinction between a button and a link but that's perhaps a weak argument.

I'll collect our options and drop them in here and we can decide which is working the best.

Reply:
Yeah, my preference would be to treat clickable things consistently, regardless of what kind of clickable thing they are. I feel like `bg-raise` does a good job of calling extra attention which (when combined with a pointer) is a doubly helpful indication.